code/obj/item/storage/backpack_belt_etc.dm

```diff
@@ -196,14 +196,14 @@
 			return
 		return ..()
 
-	attackby(obj/item/W as obj, mob/user as mob)
+	attackby(obj/item/W as obj, mob/user as mob, obj/item/storage/T)
 		if(!can_use())
 			boutput(user, "<span class='alert'>You need to wear [src] for that.</span>")
 			return
 		if (istype(W, /obj/item/storage/toolbox) || istype(W, /obj/item/storage/box) || istype(W, /obj/item/storage/belt))
 			var/obj/item/storage/S = W
 			for (var/obj/item/I in S.get_contents())
-				if (..(I, user, null, S) == 0)
+				if (..(I, user, S) == 0)
```

Comment:
Not sure why you're adding `T` as an argument but still using `S`. Pretty sure `T` is only necessary on the small storage parent's `attackby()` because for all the other objects that have a transfer interaction, you're checking and then getting the storage, not being passed the storage.

Reply:
`T` isn't relevant for `S` I think, but more to pass along to `..()` on line 210 in this case. ((tool)boxes have same structure)

Reply:
Why would you need to pass it along to `..()` though if `T` comes from `src.attackby(O, user, O.loc)`, and so is pretty much always a mob?

Reply:
Though I guess you might want to use `T` for readability since it literally says `// T for transfer - transferring items from one storage obj to another` lol.